packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -276,10 +276,10 @@ const AppointmentsForm: React.FC<AppointmentsFormProps> = ({
 
     // check if Duplicate Response Occurs
     const response: FetchResponse = await checkAppointmentConflict(appointmentPayload);
-    let errorMessage = t('appointmentDuplicateForm', 'Appointment is not available');
-    if (response?.data?.SERVICE_UNAVAILABE) {
+    let errorMessage = t('appointmentConflict', 'Appointment conflict');
```

Comment:
I changed this to something a little more generic, and made it consistent with the message in the translation file below.  "Appointment conflict" isn't the most helpful of messages, but as I understand it as it stands now we should *never* need to display this message... it would only be displayed if the Appointments module introduced a new kind of conflict checker (besides SERVICE_UNAVAILABLE & PATIENT_DOUBLE_BOOKED) and we didn't introduce a custom error message for it like we do below for those two.